app/plugins/trxs_to_mysql_service.go

```diff
@@ -38,36 +38,45 @@ func FindCreator(operation *prototype.Operation) (name string) {
 	return
 }
 
-func IsCreateAccountOp(operation *prototype.Operation) bool {
-	switch operation.Op.(type) {
-	case *prototype.Operation_Op1:
-		return true
-	default:
-		return false
-	}
+var TrxMysqlServiceName = "trxsqlservice"
+
+type TrxInfo struct {
+	ID uint64					`gorm:"primary_key;auto_increment"`
+	TrxId string `gorm:"unique_index;not null"`
+	BlockHeight uint64			`gorm:"index;not null"`
+	BlockTime uint64 `gorm:"index;not null"`
+	Invoice string `gorm:"type:longtext"`
+	Operations string `gorm:"type:longtext"`
```

Comment:
incompatible with old table design: longtext vs json

Reply:
I solved it in 37136fe